src/utils/api.js

```diff
@@ -51,3 +58,15 @@ export const postComment = (article_id, commentToPost) => {
       return data.comment;
     });
 };
+
+export const increaseCommentVote = (comment_id) => {
+  return alittApi
+    .patch(`/comments/${comment_id}`, { inc_votes: 1 })
+    .then(({ data }) => {
+      return data;
+    });
+};
+
+export const getSortBy = () => {
```

Comment:
Are you using this util? Axios params in the getArticles utils would take care of it for you